lib/rubyblok/services/get_cached_storyblok_story.rb

```diff
@@ -0,0 +1,41 @@
+module Rubyblok
```

Comment:
We don't need this class, the helper should handle if it is an API call if the environment is not cached or a database call if the environment is cached. The behaviour implemented in this class should go into the rubyblok model.

Reply:
It can be as simple as
```
def fetch_content(slug)
   find_by(slug: slug)&.content
 end
```
on the rubyblok model